.github/workflows/check-pr.yml

```
@@ -68,10 +65,5 @@
       - name: Setup Flutter with FVM
         uses: ./.github/actions/setup-fvm
       
-      - name: Get dependencies
-        working-directory: ${{ env.WORKING_DIR }}
-        run: flutter pub get
-      
       - name: Run slang analyze
-        working-directory: ${{ env.WORKING_DIR }}
-        run: dart run slang analyze
+        run: melos run analyze:slang
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)